SurfaceDuoSamples/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="surface_duo_samples_app_name">Faltbare Apps von Surface Duo</string>
```

Comment:
Here we translate the name: foldable --> Faltbare, but here: https://github.com/microsoft/surface-duo-sdk-samples-kotlin/pull/48/files#diff-0047c86a1d4c944410ae385ca6fb5626bedbbdfffff973b3c2f79ca547d17be9R10 (line 10 in this snippet), don't, we have to go for an unified approach, and imho, we shouldn't translate the app name.